.github/workflows/ci.yml

```diff
@@ -14,6 +14,7 @@ env:
   POSTGRES_USER: postgres
   POSTGRES_PASSWORD: postgres
   DATABASE_URL: "postgis://postgres:postgres@localhost/test_linkedevents"
+  FIELD_ENCRYPTION_KEYS: c87a6669a1ded2834f1dfd0830d86ef6cdd20372ac83e8c7c23feffe87e6a051
```

Comment:
Maybe you could add some default value to `/docker/django/.env.example` as well. Otherwise there will be some errors when running `python manage.py migrate registrations`'''

Or optionally mention in README.md that it's needed.

Reply:
I'd say add a default to .env.example so that it's easy to setup a dev env.

Reply:
Added to `.env.example`